fil_actor_interface/src/builtin/known_cids.yaml

```diff
@@ -0,0 +1,176 @@
+account:
```

Comment:
It'd be great to add the manifest Cid as well.

Reply:
Manifest CIDs are now available in `NETWORK_MANIFESTS`  https://github.com/ChainSafe/fil-actor-states/pull/42/files#diff-6680fa3f5d9843c6fc76003cadad09a354c54c024c8d8d68cebe58e17ada7c0eR15